vars/folioNamespaceCreate.groovy

```diff
@@ -30,6 +30,7 @@ void call(CreateNamespaceParameters args) {
     tfConfig.addVar('pg_ldp_user_password', Constants.PG_LDP_DEFAULT_PASSWORD)
     tfConfig.addVar('github_team_ids', folioTools.getGitHubTeamsIds("${Constants.ENVS_MEMBERS_LIST[args.namespaceName]},${args.members}").collect { "\"${it}\"" })
```

Comment:
there should be conditional step for eureka related helm chart...

Reply:
@eldiiar-duishenaliev 
In repsonse to your request I've added conditional helm chart deployment.
Could you please take a look?